jenkins-jobs/pipelines/build-debezium-images.groovy

```diff
@@ -7,6 +7,7 @@ STREAMS_TO_BUILD_COUNT = STREAMS_TO_BUILD_COUNT.toInteger()
 TAGS_PER_STREAM_COUNT = TAGS_PER_STREAM_COUNT.toInteger()
 GIT_CREDENTIALS_ID = 'debezium-github'
 DOCKER_CREDENTIALS_ID = 'debezium-dockerhub'
+MAIL_ID = 'jpechane@redhat.com'
```

Comment:
@ani-sha This should not be hardcoded by passed as parameter to the job, you should modify job definitions and add the param there with this value as the default one.

Reply:
Addressed it in https://github.com/debezium/debezium/pull/2765